features/homepage.feature

```diff
@@ -9,14 +9,13 @@ Feature: Homepage
     And there is a public configuration created named "Test Configuration"
     And I have a public repository named "Test Repository"
     Then I am at the homepage
-    And I should see "Home"
     And I should see "Projects"
     And I should see "Repositories"
     And I should see "Configurations"
     And I should see "Reading Groups"
     And I should see "Sign In"
     And I should see "Sign Up"
-    And I should see "Language"
+    And I should see the "en" flag
```

Comment:
What do you think about changing this step to `And I should see the flag for the current locale` and using `I18n.locale` on the step definition, instead of passing the locale as a parameter?

That way the test is more robust to future changes.